FSI/Displacement.C

```diff
@@ -2,31 +2,40 @@
 
 using namespace Foam;
 
+
 preciceAdapter::FSI::Displacement::Displacement
 (
     const Foam::fvMesh& mesh,
-    const Foam::Time& runTime,
-    int numDataLocations
+    const std::string namePointDisplacement
     /* TODO: We should add any required field names here.
     /  They would need to be vector fields.
     /  See CHT/Temperature.C for details.
     */
 )
 :
-mesh_(mesh),
+pointDisplacement_(
+    const_cast<pointVectorField*>
+    (
+        &mesh.lookupObject<pointVectorField>(namePointDisplacement)
```

Comment:
I get the following error when running the tutorial case (OpenFOAM 5, pimpleDyMFoam with velocityLaplacian):

```
request for pointVectorField pointDisplacement from objectRegistry region0 failed
    available objects of type pointVectorField are
1(pointMotionU)
```

So, in this case, this field does not exist. I assume that you are using a different mesh motion solver?

Reply:
You are correct. I adapted the case using the displacementLaplacian. This solver uses the pointDisplacement field rather than the pointMotion field. 

The zipped tutorial file is included. Note that I am using RBF instead of nearest neighbour for the displacement mapping. 

[OpenFOAM-CalculiX_dispLaplac.tar.gz](https://github.com/precice/openfoam-adapter/files/2344948/OpenFOAM-CalculiX_dispLaplac.tar.gz)

Reply:
Conclusion from our meeting: we keep this one and move to displacementLaplacian for now! (see discussion above)